README.md

```diff
@@ -39,6 +39,7 @@ Simple usage example
   ng-virtual-keyboard
   ng-virtual-keyboard-layout="alphanumeric"
   ng-virtual-keyboard-placeholder="Custom placeholder text"
+  (ng-virtual-keyboard-placeholder)="onClose()"
```

Comment:
Should this be `ng-virtual-keyboard-close` ?

Reply:
Damn these copy paste devils.